pandas/tests/api/test_api.py

```diff
@@ -41,7 +41,8 @@ class TestPDApi(Base):
     ]
 
     # these are already deprecated; awaiting removal
-    deprecated_modules = []
+    # GH #28926
+    deprecated_modules : List[str] = []
```

Comment:
Right now on master we still have 3.5 in our CI, so need to use the `# type: ...` comment syntax for compat

Reply:
fixed comment syntax. passed black and flake8. Do i need to remove GH # reference?

Reply:
Yes go ahead and remove

Reply:
done